lib/fizzbuzz.rb

```diff
@@ -1,13 +1,29 @@
 class SuperFizzBuzz
 
   def run(input)
+    output = ""
```

Comment:
Take a look at `.tap`. 
https://ruby-doc.org/core-2.5.0/Object.html#method-i-tap

Reply:
I am not seeing it. How would I use .tap here to avoid initializing output to an empty string?